### DISCUSSION TITLE
Database cluster with read/write nodes

### DISCUSSION THREAD

**Discussion by Royalsspirit** [outline/outline#6854] [Ideas]
Hello, 

first of all, thank you for this amazing project. 

We currently have a postgres cluster with Master-Slave Replication
In order to use it properly, I would like to know if it's possible to have two kind of DATABASE_URL one for write operations and another one for read operations. It will allow us Master node only for reading.

If you need for details, don't hesitate.

Thanks in advance

**Comment by tommoor** [comment]
Hi @Royalsspirit - there is no such configuration currently. Such a setup would be overkill for self hosting this project.

**Comment by Royalsspirit** [comment]
Depending of the use case, with big amount of users, it could be not that overkill. 

Thank you for your availability
